chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -392,9 +396,15 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
             props.controlProps?.widgetInstanceId ?? "");
 
         BroadcastService.getMessageByEventName(endChatEventName).subscribe((msg: ICustomEvent) => {
-            console.log("Receiving end chat event", JSON.stringify(msg.payload));
-            if (msg.payload !== uwid.current) {
+            if (msg?.payload?.runtimeId !== TelemetryManager.InternalTelemetryData.lcwRuntimeId) {
                 endChat(props, chatSDK, state, dispatch, setAdapter, setWebChatStyles, adapter, true, false, false);
+                endChatStateCleanUp(dispatch);
+                // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                (chatSDK as any).requestId = uuidv4();
+                // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                (chatSDK as any).chatToken = {};
+                // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                (chatSDK as any).reconnectId = null;
```

Comment:
How come we didn't reset these chatSDK values before, and why are they needed now?

Reply:
It’s a bug that we don’t reset the requestId. It’s causing startChatErrorPane to show up if the chat is ended by other tabs. This change is fixing the bug.